systems/subscriber/registry/pkg/client/network.go

```diff
@@ -0,0 +1,96 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	"github.com/ukama/ukama/systems/common/rest"
+)
+
+type NetworkInfoClient interface {
+	ValidateNetwork(networkID string, orgID string) error
+}
+
+type networkInfoClient struct {
+	R *RestClient
+}
+type NetworkInfo struct {
+	NetworkId     string    `json:"id"`
+	Name          string    `json:"name"`
+	OrgId         string    `json:"org_id"`
+	IsDeactivated bool      `json:"is_deactivated"`
+	CreatedAt     time.Time `json:"created_at"`
+}
+
+func NewNetworkClient(url string, debug bool) (*networkInfoClient, error) {
+
+	f, err := NewRestClient(url, debug)
+	if err != nil {
+		logrus.Errorf("Can't conncet to %s url.Error %s", url, err.Error())
+		return nil, err
+	}
+
+	N := &networkInfoClient{
+		R: f,
+	}
+
+	return N, nil
+}
+
+func (N *networkInfoClient) ValidateNetwork(networkID string, orgID string) error {
+	errStatus := &rest.ErrorMessage{}
+	network := &NetworkInfo{}
+	resp, err := N.R.C.R().
+		SetError(errStatus).
+		Get(N.R.URL.String() + "/v1/networks/" + networkID)
+
+	if err != nil {
+		logrus.Errorf("Failed to send api request to network registry. Error %s", err.Error())
+		return err
+	}
+
+	if !resp.IsSuccess() {
+		logrus.Tracef("Failed to fetch network info. HTTP resp code %d and Error message is %s", resp.StatusCode(), errStatus.Message)
+		return fmt.Errorf(" Network Info failure %s", errStatus.Message)
+	}
+
+	err = json.Unmarshal(resp.Body(), &network)
+	if err != nil {
+		logrus.Tracef("Failed to desrialize network info. Error message is %s", err.Error())
+		return fmt.Errorf("network info deserailization failure:" + err.Error())
+	} else {
```

Comment:
You can remove the else block and indent back the logrus line to the same level of the if statements.